command/aarelayer/service/aa_state.go

```diff
@@ -0,0 +1,66 @@
+package service
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/google/uuid"
+)
+
+type AATxState interface {
```

Comment:
Also some comments to this file.

Reply:
Fixed fixed https://github.com/0xPolygon/polygon-edge/pull/1252/commits/f8527534f60da34f423360164a5bfd3a2cbf29d9